Failed to compile on Fedora 32

Fedora 32 (just upgraded)
Pulled here a few hours ago.
Compile stopped after two errors:
1. julia/contrib/normalize_triplet.py", line 112, in <module>
    libgfortran_version = {
    KeyError: '1'
2. curl: (22) The requested URL returned error: 404 Not Found
    julia/deps/suitesparse.mk:144: 
    julia/deps/srccache/SuiteSparse.v5.4.0-6..tar.gz] Error 22

I pulled this morning. It still has the same errors.

> Traceback (most recent call last):
  File "/home/xijiang/Music/julia/contrib/normalize_triplet.py", line 112, in <module>
    libgfortran_version = {
KeyError: '1'
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   453  100   453    0     0   1043      0 --:--:-- --:--:-- --:--:--  1043
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404 Not Found
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404 Not Found
make[1]: *** [/home/xijiang/Music/julia/deps/suitesparse.mk:144: /home/xijiang/Music/julia/deps/srccache/SuiteSparse.v5.4.0-6..tar.gz] Error 22
make: *** [Makefile:60: julia-deps] Error 2

--- a/contrib/normalize_triplet.py
+++ b/contrib/normalize_triplet.py
@@ -110,13 +110,14 @@ def p(x):
 if libgfortran_version == "blank_libgfortran":
     if len(sys.argv) >= 3:
         libgfortran_version = {
-            "4": "libgfortran3",
-            "5": "libgfortran3",
-            "6": "libgfortran3",
-            "7": "libgfortran4",
-            "8": "libgfortran5",
-            "9": "libgfortran5",
-        }[list(filter(lambda x: re.match("\d+\.\d+(\.\d+)?", x), sys.argv[2].split()))[-1][0]]
+            "4":  "libgfortran3",
+            "5":  "libgfortran3",
+            "6":  "libgfortran3",
+            "7":  "libgfortran4",
+            "8":  "libgfortran5",
+            "9":  "libgfortran5",
+            "10": "libgfortran5",
+        }[list(filter(lambda x: re.match("\d+\.\d+(\.\d+)?", x), sys.argv[2].split()))[-1].split('.')[0]]
 
 if cxx_abi == "blank_cxx_abi":
     if len(sys.argv) == 4:

